### ISSUE TITLE
Curved Lines / Arrows

### ISSUE THREAD

**Issue by kaugummiheld** [veusz/veusz#448]
Hello guys. 

I'm bringing back a suggestion I would like to see in Veusz and stated in the [subreddit](https://www.reddit.com/r/Veusz/comments/emofms/curved_arrowslines/) a while ago. 
While it's nice that you can enhance/annotate my graphs with arrows in Veusz, straight arrows does not look as good/modern/stilish as curved ones, e.g. 
![image](https://user-images.githubusercontent.com/61326100/93482938-8bdc2c80-f900-11ea-97f6-fa5bc743533b.png)
I know, that's way more complex to implement than a straight line from point A to B, but I would really like to use this in my graphs. Maybe somebody has an idea how one could achieve this as simple as possible.

**Comment by MarkDurre** [comment]
I would like this too. Useful for indicating directions overlaying on an image

**Comment by BrainStormCenter** [comment]
Ditto! This is a great idea!

**Comment by kurthollocher** [comment]
Yes, I too would like arbitrarily curved lines (wiggly) and closed curved lines that can have interior colors, like this one that I made some time ago with old, no longer supported software.

![Image](https://github.com/user-attachments/assets/bc86c03e-faea-4a9f-9fad-f40a1ac6df02)
